### PR TITLE
Generate changelog for 2.11.3

### PR DESCRIPTION
**Pull request by sergos**
Also, remove unreleased/ entries.

NO_DOC=changelog
NO_TEST=changelog
NO_CHANGELOG=changelog

### CODE REVIEW

File: changelogs/2.11.3.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+[upgrade]: https://www.tarantool.io/en/doc/latest/book/admin/upgrades/
+
+## Functionality added or changed
+
+### Tools
+
+* Added the `--human-readable` option for the `misc.memprof` parser to print
+  sizes like 1KiB, 234MiB, 2GiB, etc.
+  Made the errors from the profilers more user-friendly (gh-9217).
```
Comment (Totktonada) [inline changelogs/2.11.3.md:37]: Let's move it to a top-level section `## Tools`.

File: changelogs/2.11.3.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+2.11.3 is the next stable release in the [long-term support (LTS)
+version][release_policy][release_policy] 2.11.x release series.
+
+The label "stable" means there are 1.10.x-based applications running in
+production for quite a while without known crashes, incorrect results or
+other showstopper bugs.
+
+This release introduces 1 improvements and resolves roughly 54
+issues since the 2.11.2 version.
```
Comment (Totktonada) [inline changelogs/2.11.3.md:16]: This '1 improvements' is about tools and I think we'll have only bugfixes here after moving it to a top-level `## Tools` section.

**Review by Totktonada — APPROVED** [review]
I've no objections.

File: changelogs/2.11.3.md (hunk 1 above)
Comment (sergos) [reply changelogs/2.11.3.md]: done.

File: changelogs/2.11.3.md (hunk 2 above)
Comment (sergos) [reply changelogs/2.11.3.md]: done

**Pull request merged**